BUILD

```diff
@@ -1,13 +1,37 @@
 package(default_visibility = ["//visibility:public"])
 
-ALL_MODULES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ASMCLI_MODULES = glob(["asmcli/**"])
+NODE_MODULES = glob(["node_modules/**"])
+ASMCLI_SOURCES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ALL_UNIT_TESTS = glob(["asmcli/tests/*.bats"])
 MERGE_OUT = ["asmcli"]
 
 genrule(
     name = "merge",
-    srcs = ALL_MODULES,
+    srcs = ASMCLI_SOURCES,
     outs = MERGE_OUT,
     cmd = "$(location scripts/release-asm/merge) $(OUTS) $(SRCS)",
     tools = ["scripts/release-asm/merge"],
     executable = True,
 )
+
+sh_library(
+    name = "asmcli_modules",
+    data = ASMCLI_MODULES,
+)
+
+sh_library(
+    name = "node_modules",
+    data = NODE_MODULES,
+)
+
+sh_test(
+    name = "unit_test",
+    size = "small",
+    srcs = ["scripts/tests/run_tests"],
```

Comment:
Instead of having a specific script, can we just do something like

```
sh_test(
  name = "test",
  ...
  env = {"SOURCE_FILES" : ASMCLI_SOURCES},
  srcs = ["node_packages/bats/bin/bats"],
  args = ALL_UNIT_TESTS
```

Reply:
This is way too smart. Will experiment to see if it works. Thanks!